commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/MiniCartTag.java

```diff
@@ -96,8 +98,20 @@ public int doStartTag() {
 
 			putValue("detailsUrl", detailsURL);
 
-			putValue("isDisabled", false);
-			putValue("isOpen", false);
+			boolean disabled = false;
+
+			Layout layout = themeDisplay.getLayout();
+
+			Layout checkoutLayout =
```

Comment:
@alecsloan To identify if exist a checkout page or not we can't do this, use CommerceOrderHttpHelper.getCommerceCheckoutPortletURL

Reply:
I'll have to make a Util wrapper, that's ok?

Reply:
This isn't for checking if the checkout page exists. It's comparing the layout of the current page to the checkout page's. Do you not like that we're explicitly looking for the "/checkout" friendly url?

Reply:
Yeah the "/checkout" URL being the checkout page may not hold true for a custom site using Commerce, so we should do this a different way.

Reply:
Where do we stand with this? Is the MiniCart usable on non-minium sites or is it like the Account Selector where we don't really support use on custom sites?